### DISCUSSION TITLE
Pull request

### DISCUSSION THREAD

**Discussion by vijaychandra1910** [recodehive/awesome-github-profiles#537] [Q&A] [answered]
I am unable to rend pill request

**Comment by sanjay-kv** [comment]
sorry i didnt get you

**Comment by vijaychandra1910** [comment]
are any bounding timing to complete assigning project??

**Reply by sanjay-kv** [reply]
No, if keeping it stale will be assign to different user.

**Comment by vijaychandra1910** [comment]
I will get points on getting assigned or completing the project.

**Reply by sanjay-kv** [reply]
you wont get points for getting assinged. only if it get merged points will be given

**Comment by vijaychandra1910** [comment]
thanks

**Comment by trahulprabhu38** [comment, marked as answer]
i made a PR which got merged, but it doesnt have any tags of level and gssoc , what to do.

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/2a6c9191-232f-4797-9edd-b1b4bce7f9fb)
you can see the red thing, as there were 3500 files changed, ]
